south-african-journal-of-animal-science.csl

```diff
@@ -88,17 +88,12 @@
         <group delimiter=" ">
           <text value="Available at"/>
           <text variable="URL"/>
-          <group prefix="(verified " suffix=")">
             <date variable="accessed">
               <date-part name="day" suffix=" "/>
               <date-part name="month" suffix=" "/>
               <date-part name="year"/>
             </date>
           </group>
-        </group>
-      </if>
-    </choose>
-  </macro>
```

Comment:
@Yusufras, this isn't right. You deleted several closing tags that were necessary.